server/cli/src/plugins.rs

```diff
@@ -16,11 +8,22 @@ use reqwest::Url;
 use serde::Deserialize;
 use serde_json::json;
 use service::backend_plugin::plugin_provider::PluginBundle;
+use std::{
+    ffi::OsStr,
+    fs,
+    path::{Path, PathBuf},
+    process::Command,
+};
 
 use thiserror::Error as ThisError;
 use util::format_error;
+
+use crate::{
+    queries_mutations::INSTALL_PLUGINS, run_command_with_error, Api, ApiError, CommandError,
+};
+
 #[derive(ThisError, Debug)]
-pub(super) enum Error {
+pub enum Error {
```

Comment:
So all of these visibility changes were due to changing plugins to be exported/imported through lib and use in CLI, i am still confused, sorry for diff